lib/secure_headers/view_helper.rb

```diff
@@ -7,21 +7,45 @@ module ViewHelpers
     class UnexpectedHashedScriptException < StandardError; end
 
     # Public: create a style tag using the content security policy nonce.
-    # Instructs secure_headers to append a nonce to style/script-src directives.
+    # Instructs secure_headers to append a nonce to style-src directive.
     #
     # Returns an html-safe style tag with the nonce attribute.
     def nonced_style_tag(content_or_options = {}, &block)
       nonced_tag(:style, content_or_options, block)
```

Comment:
Should this be a passthrough to `nonced_stylesheet_link_tag` instead?

Reply:
No, `nonced_style_tag` generates `<style>` whereas `nonced_stylesheet_link_tag` generates a `<link />`